extensions/http-curl/protocols/RESTReceiver.cpp

```diff
@@ -65,38 +65,8 @@ void RESTReceiver::initialize(const std::shared_ptr<core::controller::Controller
   }
 }
 int16_t RESTReceiver::heartbeat(const C2Payload &payload) {
-  std::string operation_request_str = getOperation(payload);
```

Comment:
I assume serializeJsonRootPayload has evolved to nest the payloads? Have you tested this with a C2 server? We should do that before merging.

Reply:
All of the tests passed, and I made sure the JSON payloads generated by the exchanges were identical. But no, I haven't tested it in a production setting.